openmetadata-ui/src/main/resources/ui/src/jsons/en.ts

```diff
@@ -179,6 +179,7 @@ const jsonData = {
     'test-connection-success': 'Connection tested successfully!',
 
     'user-restored-success': 'User restored successfully!',
+    'team-restored-success': 'Team restored successfully!',
```

Comment:
```suggestion
    'entity-restored-success': '{entity} restored successfully!',
```
Let's use generic so that we don't end up with multiple localization strings

Reply:
@chirag-madlani Updated!